docs/clients/p4rt-ctl.rst

```diff
@@ -469,6 +469,86 @@ Example:
 
    p4rt-ctl dump-entries br0
 
+Add configuration to meter table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+   add-meter-config SWITCH METER_TBL METER_CONFIGURATION
+
+Arguments:
+
+* ``SWITCH``: Bridge name. Maps internally to device name.
+* ``METER_TBL``: Meter table.
+* ``METER_CONFIGURATION``: COnfiguration for meter table.
+
+Example:
+
+.. code-block:: bash
+
+   p4rt-ctl add-meter-config br0 my_control.meter1 "meter_id=2244878476,meter_index=10,meter_config=policer_meter_prof_id=0,policer_spec_cir_unit=1,policer_spec_cbs_unit=1,policer_spec_eir_unit=1,policer_spec_ebs_unit=1,policer_spec_cir=1000,policer_spec_cbs=1500,policer_spec_eir=1000,policer_spec_ebs=1500"
+
+Add table entry (rule) for Direct meter
```

Comment:
```suggestion
Add table entry (rule) for direct meter
```
The uppercase `D` here stands out in an odd way. Other things like indirect meter in this document are not capitalized, so should switch to lowercase d.